user/urls.py

```diff
@@ -1,5 +1,5 @@
 from django.conf.urls import url, patterns, include
-from views import PublicUserProfile, ProfileListView
+from views import PublicUserProfile, ProfileListView, oklogin, login_redirect_opensubs
```

Comment:
IMO, `oklogin` is confusing. Just like we have `main`, we should have a `login` view. 
Also `login_redirect_opensubs` can be just `login_redirect`. It can be used by many more apps, not just opensubs